backend/src/apis/routes/CMT_zip_file.py

```diff
@@ -0,0 +1,29 @@
+import os
+import zipfile
+from fastapi import FastAPI, APIRouter, HTTPException
+from fastapi.responses import FileResponse
+from pydantic import BaseModel
+from apis.utils.directory import dire
+
+router = APIRouter()
+
+class ModelName(BaseModel):
+    modelname: str
+
+@router.post("/zip_model/")
+async def zip_model(model: ModelName):
+    model_base_path = os.path.join(dire.temp_path, model.modelname)
+
+    txt_file_path = f"{model_base_path}.txt"
```

Comment:
Filter out the datetime from the model name to save as just item type.

Reply:
The issue is already resolve in the commit [f83a839](https://github.com/cljiahao/csam_ai_training/pull/45/commits/f83a839f3f19b53e43a3b3120426d2e370a385d8)